web/assets/js/main_out.js

```diff
@@ -1390,88 +1445,57 @@
         window.requestAnimationFrame(drawGame);
+        log.info(`init done in ${Date.now() - LOAD_START}ms`);
     }
-    wHandle.setserver = function(arg) {
+    window.setserver = function(arg) {
         if (wsUrl === arg) return;
         wsInit(arg);
     };
-    wHandle.setDarkTheme = function(a) {
-        settings.darkTheme = a;
-        drawStats();
-    };
-    wHandle.setShowMass = function(a) {
-        settings.showMass = a;
-    };
-    wHandle.setSkins = function(a) {
-        settings.showSkins = a;
-    };
-    wHandle.setColors = function(a) {
-        settings.showColor = !a;
-    };
-    wHandle.setNames = function(a) {
-        settings.showNames = a;
-        drawLeaderboard();
-    };
-    wHandle.setChatHide = function(a) {
-        settings.showChat = !a;
-        drawChat();
-    };
-    wHandle.setMinimap = function(a) {
-        settings.showMinimap = !a;
-    };
-    wHandle.setBorders = function(a) {
-        settings.showBorders = a;
-    };
-    wHandle.setPosition = function(a) {
-        settings.showPosition = a;
-    };
-    wHandle.setShowGrid = function(a) {
-        settings.showGrid = !a;
-    };
-    wHandle.setSounds = function(a) {
-        settings.sounds = a;
-        volumeInput.style.display = a ? "inline" : "none";
-    };
-    wHandle.spectate = function(a) {
+    window.spectate = function(a) {
         wsSend(UINT8_CACHE[1]);
         stats.maxScore = 0;
         hideESCOverlay();
     };
-    wHandle.play = function(a, b) {
-        sendPlay((!a ? b : '{'+a+'}'+b));
-        hideESCOverlay();
+    window.changeSkin = function(a) {
+        byId("skin").value = a;
+        settings.skin = a;
+        byId("gallery").hide();
     };
-    wHandle.changeSkin = function(a) {
-      byId('skin').value = a;
-    };
-    wHandle.openSkinsList = function() {
-        if (wjQuery("#inPageModalTitle").text() === "Skins") return;
-        wjQuery('#inPageModalTitle').text('Skins');
-        if(typeof allSkins == 'undefined') {
-          wjQuery.get("include/gallery.php").then(function(data) {
-              wjQuery("#inPageModalBody").html(data);
-          });
-        } else {
-          var galleryHtml = '<link href="assets/css/gallery.css" rel="stylesheet"><div class="row center"><ul>' + allSkins.map(name => '<li class="skin" onclick="changeSkin($(this).find(\'.title\').text())" data-dismiss="modal"><div class="circular" style=\'background-image: url("' + './skins/' + name + '.png")\'></div><h4 class="title">' + name + '</h4></li>').join('') + '</ul></div>';
-          byId('inPageModalBody').innerHTML = galleryHtml;
-        }
+    window.openSkinsList = function() {
+        buildGallery();
+        byId("skinsUL").style.height = window.innerHeight * 0.75 + "px";
+        byId("gallery").show();
     };
-    wHandle.addEventListener("DOMContentLoaded", init);
-})(window, window.jQuery);
+    window.addEventListener("DOMContentLoaded", init);
```

Comment:
why not use onload event?

Reply:
Preference thing, It's also possible a userscript could set the property back to undefined, stopping `init` from running.

Reply:
It's not possible to remove an addEventListener without having any info about it

Reply:
That's why it's better than setting the `onload` property.

Reply:
You can addEventListener load event too...